python-opendata/opendata/utils.py

```diff
@@ -0,0 +1,6 @@
+import arrow
```

Comment:
just for a single call, this is costly to have an extra dependency :)

Reply:
The mental pain that I experience every time I have to work with Python's builtin `strftime`/`strptime` is also costly...
But as this is probably the only place where we will use this I am open to a PR that removes this dependency. ;)